genai-perf/genai_perf/export_data/telemetry_data_exporter_util.py

```diff
@@ -59,35 +59,47 @@
 }
 
 
-def merge_telemetry_stats_json(
-    telemetry_stats: Optional[Dict], stats_and_args: Dict
-) -> None:
-    if telemetry_stats is not None:
-        stats_and_args.update({"telemetry_stats": telemetry_stats})
+def merge_telemetry_stats_json(telemetry_stats: Dict, stats_and_args: Dict) -> None:
+    filtered_telemetry_stats = {
```

Comment:
Checks if there are any Telemetry statistics collected by filtering for any actual data other than the unit name (which is there by default).